src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapFullScenario.scala

```diff
@@ -0,0 +1,67 @@
+package com.linagora.gatling.imap.scenario
+
+import java.util.Calendar
+
+import com.linagora.gatling.imap.PreDef._
```

Comment:
You know, Scala is the kingdom of implicits :)
More seriously that's because it's a DSL, so you expect to have all commands available.